loader/src/main/java/net/neoforged/fml/common/asm/RuntimeEnumExtender.java

```diff
@@ -34,6 +38,21 @@ public class RuntimeEnumExtender implements ILaunchPluginService {
     private final String CLEAN_DESC = Type.getMethodDescriptor(Type.VOID_TYPE, Type.getType(Class.class));
     private final String NAME_DESC = Type.getMethodDescriptor(STRING);
     private final String EQUALS_DESC = Type.getMethodDescriptor(Type.BOOLEAN_TYPE, STRING);
+    private final Type LIST_TYPE = Type.getType(List.class);
+    private final Type ARRAY_LIST_TYPE = Type.getType(ArrayList.class);
+    private final String ARRAY_LIST_INIT_DESC = Type.getMethodDescriptor(Type.VOID_TYPE, Type.getType(Collection.class));
+    private final String VANILLA_FIELDS_SIGN = "Ljava/util/List<LMyExtensibleEnum;>";
```

Comment:
```suggestion
    private final String VANILLA_FIELDS_SIGN = "Ljava/util/List<%s>;";
```
JVM doesn't really care about signatures, but lets make it right because we can :)

Reply:
yeah, it doesn't really care, but can you point me to the place in the spec where %s is valid as I didn't see that

Reply:
I believe this suggestion would be read alongside their other suggestion, which calls `formatted` on this string with the appropriate descriptor.